src/main/java/io/vertx/core/shareddata/SharedData.java

```diff
@@ -39,11 +39,21 @@
    *
    * @param name  the name of the map
    * @param resultHandler  the map will be returned asynchronously in this handler
+   * @throws IllegalStateException if the parent {@link io.vertx.core.Vertx} instance is not clustered
    */
   <K, V> void getClusterWideMap(String name, Handler<AsyncResult<AsyncMap<K, V>>> resultHandler);
 
   /**
-   * Get a cluster wide lock with the specified name. The lock will be passed to the handler when it is available.
+   * Get the {@link AsyncMap} with the specified name. When clustered, the map is accessible to all nodes in the cluster
```

Comment:
Add a note about cluster wide latency like it is in doc

Reply:
Done